iocage/lib/Config/Jail/BaseConfig.py

```diff
@@ -114,9 +114,13 @@ def clone(
             if (key in ["id", "name", "uuid"]) and (current_id is not None):
                 value = current_id
 
-            self.__setitem__(key, value, skip_on_error=skip_on_error)
+            self.__setitem__(  # noqa: T484
```

Comment:
how come this isn't necessary with other `__setitem__` function(call?)s?

Reply:
This __setitem__ function extends the method with the skip_on_error argument. MyPy thinks it is invalid to do so.